conftest.py

```diff
@@ -8,9 +8,9 @@
 
 
 # Specs of "clean" (uninitialized) databases
-CLEAN_DATABASE_SPECS = list(kcidb.db.misc.parse_spec_list(
-    os.environ.get("KCIDB_CLEAN_TEST_DATABASES", "")
-))
+CLEAN_DATABASE_SPECS = list(
+    kcidb.db.misc.parse_spec_list(os.environ.get("KCIDB_CLEAN_TEST_DATABASES", ""))
+)
```

Comment:
Hmm, does `pydocstyle` actually request **code** changes? If so, please explain that in the commit message. If not, and you think this should be done for some other reason, please move this and all other code changes to a separate commit, and explain the purpose and reason for the change in the commit message.

Reply:
No, it doesnt. Actually the large amount changes are of the Reformatting done by me as an extra to make it look more good and readable using a formatting extension in vs code.
Should I move all these formatting changes to separate commit or raise a separate issue reg it and then make a PR for that?

Reply:
I appreciate the effort to make the code look nicer. However, please keep **all** reformatting of otherwise-unchanged code in separate commits, and explain why you're doing them in the commit messages. Then we can discuss them. Otherwise they make reviewing harder. 

Please also note that some of your changes take lines over the 79-column limit, and so shouldn't be done, at least in that shape.

Thank you :heart:

Reply:
> I appreciate the effort to make the code look nicer. However, please keep **all** reformatting of otherwise-unchanged code in separate commits, and explain why you're doing them in the commit messages. Then we can discuss them. Otherwise, they make reviewing harder.
> 
> Please also note that some of your changes take lines over the 79-column limit, and so shouldn't be done, at least in that shape.
> 
> Thank you heart

Yes they are, actually the thing is I'm a bit lost cuz if I try to maintain them under the 78 limit then the pydoc creates the issue like the sentence should end with '.' not with other alphabets and then sometimes gets an error message of blank space needed after first-line description. So this is happening.

Reply:
Let's treat the docstrings and the code separately. This PR should only concern itself with docstrings, since it's adding the `pydocstyle` to CI. We can discuss and deal with code formatting separately.

Pydocstyle demands that a multi-line docstring should have a summary line, and it **must** be a single line. Currently the code often has first paragraphs (or several) just containing the description of the function/class, and there is **no** summary line as such. Some (most?) of those are short enough to be on a single line, and pydocstyle would count them as "summaries". Some of the others are longer. So, if you want to do this change, you have to come up with a short summary line for those, and put it above the existing description. However, consider these rules, and use your wits to do what makes best sense, it might not always work out like this. This is a documentation-writing exercise to a considerable extent, not only code and text manipulation.

Lastly, read the [PEP 257](https://peps.python.org/pep-0257/) to understand the complete requirements pydocstyle is trying to enforce.

If you'd like we could have a video call about this and other issues, e.g. tomorrow. Just tell me which times you're available, on Slack, and I'll try to find 30-60 minutes.

Reply:
Okay